app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -66,6 +66,8 @@ def start
     end
 
     queue_signal(:pod_wait)
+  rescue
+    queue_abort("unexpected exception, see log")
```

Comment:
@blomquisg I forgot I wanted to raise the unexpected errors. 
That way I can have them logged just as they are logged today (only the cleanup was missing).
Does that make sense?